stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/LinkAttachmentFactory.kt

```diff
@@ -44,15 +50,20 @@ public class LinkAttachmentFactory : AttachmentFactory(
  *
  * @param attachmentState - The state of the attachment, holding the root modifier, the message
  * and the onLongItemClick handler.
+ *
+ * @param descriptionLineLimit - The limit of how many lines we show for the link description.
  * */
 @OptIn(ExperimentalFoundationApi::class)
 @Composable
-public fun LinkAttachmentContent(attachmentState: AttachmentState) {
+public fun LinkAttachmentContent(
+    attachmentState: AttachmentState,
+    descriptionLineLimit: Int,
```

Comment:
What about calling this something that has `maxLines` in it? Would be a familiar name for Android devs, both `TextView` and now `Text` use that as the name for this value.